devtools/config-doc-maven-plugin/src/main/java/io/quarkus/maven/config/doc/generator/AbstractFormatter.java

```diff
@@ -79,11 +79,11 @@ public String formatTypeDescription(ConfigProperty configProperty, Context conte
                                     .transform(javadocElement.get().description(), javadocElement.get().format(),
                                             javadocFormat()));
                         })
-                        .collect(Collectors.joining(", "));
+                        .collect(Collectors.joining("`\n- `", "- `", "`"));
```

Comment:
The elements are already backticked here (because there might be some tooltip infra) so you just need to apply the list formatting, no backticks.

Reply:
Ah good catch, fixed